Acr.XamForms.SignaturePad.iOS/SignatureServiceView.cs

```diff
@@ -1,62 +1,53 @@
 using System;
 using CoreGraphics;
 using UIKit;
-using NativeView = global::SignaturePad.SignaturePadView;
+using NativeView = SignaturePad.iOS.SignaturePadView;
 
 
-namespace Acr.XamForms.SignaturePad.iOS {
+namespace SignaturePad.iOS
+{
+	public class SignatureServiceView : UIView
+	{
+		public NativeView Signature { get; set; }
+		public UIButton SaveButton { get; private set; }
+		public UIButton CancelButton { get; private set; }
 
-    public class SignatureServiceView : UIView {
 
-        public NativeView Signature { get; set; }
-        public UIButton SaveButton { get; private set; }
-        public UIButton CancelButton { get; private set; }
+		public SignatureServiceView()
+		{
+			//BackgroundColor = UIColor.White;
+			this.SaveButton = UIButton.FromType(UIButtonType.RoundedRect);
+			this.CancelButton = UIButton.FromType(UIButtonType.RoundedRect);
+			this.Frame = (CGRect)UIScreen.MainScreen.ApplicationFrame;
 
+			this.Signature = new NativeView();
+			this.TranslatesAutoresizingMaskIntoConstraints = false;
 
-        public SignatureServiceView() {
-            //BackgroundColor = UIColor.White;
-            this.SaveButton = UIButton.FromType(UIButtonType.RoundedRect);
-            this.CancelButton = UIButton.FromType(UIButtonType.RoundedRect);
-            this.Frame = (CGRect)UIScreen.MainScreen.ApplicationFrame;
+			this.AddSubview(this.Signature);
+			this.AddSubview(this.SaveButton);
+			this.AddSubview(this.CancelButton);
+		}
 
-            this.Signature = new NativeView();
-            this.TranslatesAutoresizingMaskIntoConstraints = false;
 
-            this.AddSubview(this.Signature);
-            this.AddSubview(this.SaveButton);
-            this.AddSubview(this.CancelButton);
-        }
+		public override void LayoutSubviews()
+		{
+			var frame = UIScreen.MainScreen.ApplicationFrame;
```

Comment:
Originally using var frame = (CGRect)this.Frame 
would cause the size to shrink everytime on rotation since you would be calling the previously calculated frame and calculate off of that frame. You would only notice this if you would rotate the device a couple of times. Setting this to the application frame means that we will always be working with a fresh frame size.